### PR TITLE
dummy-pull-request

### PR DESCRIPTION
**Pull request by samuelstroschein**
Testing whether pull requests lose context if files are moved. See https://github.com/inlang/inlang/pull/351#issuecomment-1418190958

<a href="https://gitpod.io/#https://github.com/inlang/inlang/pull/358"></a>

### CODE REVIEW

**Comment by samuelstroschein** [comment]
@ivanhofer can you make some comments?

File: source-code/core/src/lint/with-cache.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+import type * as ast from "../ast/index.js";
+
+// ------------------------------------------------------------------------------------------------
+// ast extensions
+
+type WithCache = {
+  cache: {
+    index: 5;
```
Comment (ivanhofer) [inline source-code/core/src/lint/with-cache.ts:12]: I would choose `4` here

File: source-code/core/src/lint/with-lint.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+// lint types
+
+export type LintResult = {
+  id: string;
+  type: "warning" | "error";
+  message: string;
+};
+
+// ------------------------------------------------------------------------------------------------
+/// tests
```
Comment (ivanhofer) [inline source-code/core/src/lint/with-lint.ts:37]:
```suggestion
// tests
```

**Review by ivanhofer — APPROVED** [review]
looks good to me!

**Comment by ivanhofer** [comment]
> @ivanhofer can you make some comments?

Just added a few comments

File: source-code/core/src/lint/with-cache.ts (hunk 1 above)
Comment (samuelstroschein) [reply source-code/core/src/lint/with-cache.ts:12]: Haha, totally agree!

**Pull request closed without merging**